src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,10 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    int productivity;
+
     public Bulldozer() {
+        productivity = 1000;
```

Comment:
As for me, you can do inizealize object in BulldozerProducer  in special method.

Reply:
same for Truck and Excavator

Reply:
done